components/script/dom/event.rs

```diff
@@ -297,6 +297,18 @@ impl EventMethods for Event {
         self.cancelable.get()
     }
 
+    // https://dom.spec.whatwg.org/#dom-event-cancelbubble
+    fn CancelBubble(&self) -> bool {
+        self.stop_propagation.get()
+    }
+
+    // https://dom.spec.whatwg.org/#dom-event-cancelbubble
```

Comment:
Nit: Link points to the attribute's getter instead of the setter.

Reply:
Ah, looks like it's impossible to link to the setter instead of the getter. In that case, we should just leave it as `https://dom.spec.whatwg.org/#dom-event-cancelbubble`.

Reply:
Thanks!